packages/renderer-aws-lambda/src/index.spec.js

```diff
@@ -0,0 +1,9 @@
+const mod = require('./index');
```

Comment:
Just out of curiosity 😬 

What does `mod` mean? 🤔

Reply:
`module`, but it's a reserved word, so you can't/shouldn't have a variable with that name.